scripts/move_s3_release.py

```diff
@@ -26,42 +26,49 @@
 if __name__ == '__main__':
     boto_path = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../packages/boto-2.28.0-py2.7.egg'))
     sys.path.insert(0, boto_path)
-    usage = '''usage: %prog [options] command(s)
+    usage = '''usage: %prog [options] command
 
 Commands:
-move_release - Move a release on S3 to a new channel
-
-Multiple commands can be specified
+move_release     - Move a release on S3 to a new channel
+redirect_release - Redirect a release on S3 to a new channel
 '''
     parser = optparse.OptionParser(usage)
 
     parser.add_option('--sha1', dest='sha1',
                       default = None,
-                      help = 'SHA1 of the release to move')
+                      help = 'SHA1 of the source release to move or redirect')
 
     parser.add_option('--to-channel', dest='to_channel',
                       default = None,
-                      help = 'Target channel to move release to')
+                      help = 'Target channel to move or redirect the release to')
 
     default_archive_path = CDN_UPLOAD_URL
     parser.add_option('--archive-path', dest='archive_path',
                       default = default_archive_path,
                       help = 'S3 path where archives are uploaded')
 
-    options, args = parser.parse_args()
+    options, commands = parser.parse_args()
 
-    if not args:
+    if len(commands) != 1:
```

Comment:
We no longer support multiple command, but I don't think that made sense for the script earlier either?